src/components/InputField.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 const InputField = ({ inputProps, labelProps }) => {
   return (
-    <label className={labelProps.className}>
+    <label className={labelProps.className} aria-label={labelProps.ariaLabel}>
```

Comment:
I think I just thought of adding it in case we have empty labels, like in `JoinAShoppingList` and the search input. I'm still reading on the subject so we'll probably need to change a few things later. 😸

Reply:
Hmm this is interesting. I looked it up as well. Technically you can add `aria-label` to situations where a label isn't visible. This [page](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html) might be helpful.

Reply:
Thank you for the link, Pheobe! :))